roles/openshift_health_checker/test/openshift_check_test.py

```diff
@@ -38,3 +38,24 @@ def test_get_var_error(task_vars, missing_keys):
 def test_get_var_default(task_vars, missing_keys):
     default = object()
     assert get_var(task_vars, *missing_keys, default=default) == default
+
+
+@pytest.mark.parametrize("release,expected", [
+    ('1.2', '1.2'),
+    (1.2, '1.2'),
+    ('v1.2', '1.2'),
+    ('v1.2.3.4', '1.2.3.4'),
```

Comment:
Sounds good. Ah, the empty input would cause an error elsewhere, but it's still good to check here. I wasn't sure if you thought the input validation/assertions should go here or in the modules that use the methods, but I'm thinking here.